openpype/plugins/publish/integrate_hero_version.py

```diff
@@ -411,9 +407,21 @@ def integrate_instance(
                 # Create representation
                 else:
                     repre.pop("_id", None)
+                    repre["_id"] = ObjectId()
                     op_session.create_entity(project_name, "representation",
                                              repre)
```

Comment:
Please don't use `ObjectId` in code if possible.
```suggestion
                    repre["_id"] = ObjectId()
                    operation = op_session.create_entity(
                        project_name, "representation", repre)
                    repre["_id"] = operation.entity_id
```

Reply:
👆